src/kernel-install/kernel-install.in

```diff
@@ -108,15 +108,17 @@ initrd_generator=
 _MACHINE_ID_SAVED="$MACHINE_ID"
 _BOOT_ROOT_SAVED="$BOOT_ROOT"
 
-if [ -f "/etc/kernel/install.conf" ]; then
```

Comment:
wouldn't it be easier to just always set KERNEL_INSTALL_CONF_ROOT if not set already, and then always use that var? i.e. add:

```sh
if [ -z "$KERNEL_INSTALL_CONF_ROOT" ]; then
        KERNEL_INSTALL_CONF_ROOT=/etc/kernel
fi
```
That should make things easier when referencing the dir, because we can then always just use the var

Reply:
The variable is used for multiple files, and they need different logic. E.g. for `install.conf` we read /etc/ and /usr/lib/, but for `tries`, only `/etc/`. So I think it's better to keep it as is.